app/src/main/res/values/strings.xml

```diff
@@ -1686,7 +1686,7 @@
     <string name="places_toggle_buttons_map">Map</string>
     <string name="places_toggle_buttons_list">List</string>
     <string name="places_nav_tab_tooltip_message">Discover Wikipedia articles around you with Places</string>
-    <string name="places_article_menu_tooltip_message"><![CDATA[Tap <b>\'View on map\'</b> to explore this article\'s location]]></string>
+    <string name="places_article_menu_tooltip_message"><![CDATA[Tap \'View on map\' to \nexplore this article\'s location]]></string>
```

Comment:
Do we actually need an explicit newline `\n` there? Shouldn't we allow the text to overflow normally?

Reply:
It makes the tooltip very wide like this, so I had added line-break to get closer to the designs:
![long_tooltip](https://github.com/wikimedia/apps-android-wikipedia/assets/9540770/f6a51c25-8059-4acb-97d7-40205ea6c82f)